Packs/CommonPlaybooks/ReleaseNotes/2_3_47.md

```diff
@@ -0,0 +1,9 @@
+
+#### Playbooks
+
+##### Block IP - Generic v3
+
+- Input `DAGName` was added. This input determines whether Palo Alto Networks Panorama or Firewall Dynamic Address Groups are used.
+Determine the Dynamic Address Group name for IPs list handling.
+- Change the following task input: `address_group_name` on the task 'PAN-OS DAG Configuration'
```

Comment:
The input's value of the `address_group_name` argument was changed in the 'PAN-OS DAG Configuration' task.